pages/authentication.css

```diff
@@ -0,0 +1,42 @@
+@import url('/style.css');
+@import url('/pages/products.css');
+.form-grp{
+    display: flex;
+    flex-direction: column;
+    padding: 1rem;
+    justify-content: center;
+    margin: auto;
+}
+.form-grp>input{
+    font-size: 1.5rem;
+    outline: none;
+    border: 2px solid var(--cta);
+    border-radius:0.3rem ;
+    margin:1rem;
+    
+}
+.display{
+    max-width: fit-content;
+}
+.form-grp>button{
+    margin: 1rem auto;
+}
+.form-grp>a{
+    text-decoration: none;
+}
+.form-accept{
+    display: flex;
+    justify-content: space-between;
```

Comment:
This CSS style rule is being applied on your Signup page (https://deploy-preview-4--nectar-decor.netlify.app/pages/signup.html) and hence giving too much space between the checkbox and the `accept terms and conditions` text. Please fix this minor issue.

Reply:
Thank you for pointing it out Devanshu I have made the changes and will merge the pr